docs/lib/menu.json

```diff
@@ -15,6 +15,7 @@
     "utilities",
     "client-configuration",
     "debugging",
+    "info",
```

Comment:
hhhm maybe "Privacy policy" or something more explicit?

Reply:
Nit: This is not a policy, this is a description.

Reply:
Keeping with Info for now. Will update this later if required.